source/github-helpers/api.ts

```diff
@@ -103,7 +105,7 @@ export const v3 = mem(async (
 	const {personalToken} = await settings;
 
 	if (query.startsWith('/')) {
-		throw new TypeError('The query parameter must not start with a slash.');
+		query = `repos/${getRepoURL()}${query}`;
```

Comment:
It should be the opposite. `/repos` marks the root while `${query}` would be relative to the current repo.

```js
	if (query.startsWith('/')) {
		query = query.slice(1);
	} else {
 		query = `repos/${getRepoURL()}${query}`;
```

Also please please avoid using template literals unless there are multiple strings being interspersed. Look how much more understandable this is without having to figure out if there's any significant character in the sea of `${}${}`

```js
 		query = 'repos/' + getRepoURL() + query;
```